vm/isolated/isolated.go

```diff
@@ -51,12 +54,18 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if err := config.LoadData(env.Config, cfg); err != nil {
 		return nil, err
 	}
+	if cfg.Host == "" {
+		return nil, fmt.Errorf("config param Host is empty")
+	}
 	if len(cfg.Targets) == 0 {
 		return nil, fmt.Errorf("config param targets is empty")
 	}
 	if cfg.TargetDir == "" {
 		return nil, fmt.Errorf("config param target_dir is empty")
 	}
+	if cfg.USBDeviceNum == "" {
```

Comment:
If cfg.USBDeviceNum is empty and cfg.TargetReboot is true,  it causes error in repair().
So, how about this?:
	if cfg.USBDeviceNum == "" && cfg.TargetReboot == true {
           return nil, fmt.Errorf("config param target_reboot should be 'false' when usb_device_num is empty")
        }

Reply:
and, actually I'm not familiar with github, 
should I change my code and sent PR again?
just like I did this PR?

Reply:
`TargetReboot` should work without `USBDeviceNum` as it worked before. With both `TargetReboot` and `USBDeviceNum` your new code should be executed.

Reply:
Just add more commits to the branch that you've used and push it. Optionally, you can amend/squash and do a force push to keep a single commit.

Reply:
Actually, I think the previous repair() function cannot reboot the target machine.
It used ssh command to reboot, but it is impossible 
because if kernel is crashed by fuzzing, the host cannot ssh to the target.

So, I think `TargetReboot` is meaningless without `USBDeviceNum`.

Reply:
It depends on the kernel crash. If it's just a WARNING, then the ssh will work fine and reboot the machine. Let's not change the current behavior when `USBDeviceNum` is not enabled.

Reply:
Oh. I didn't  know the case. You are right. I committed again.